Dockerfile.nginx

```diff
@@ -0,0 +1,31 @@
+
+FROM ezsystems/php:7.1-v1 as app
+
+ENV SYMFONY_ENV=prod
+
+# Copy in project files into work dir
+COPY . /var/www
+
+# assetic:dump requires parameters.yml, so let's generate it
+RUN composer run-script build --no-interaction
```

Comment:
`build` => `post-install-cmd`, safer for future changes. Symfony Flex uses `symfony-scripts`, goes for other places as well